src/blocks/xrandr.rs

```diff
@@ -194,13 +195,16 @@ impl Xrandr {
 
     fn display(&mut self) -> Result<()> {
         if let Some(m) = self.monitors.get(self.current_idx) {
-            let values = map!("{display}" => m.name.clone(),
-                              "{brightness}" => m.brightness.to_string(),
-                              "{brightness_icon}" => self.shared_config.get_icon("backlight_full").unwrap_or_default().trim().to_string(),
-                              "{resolution}" => m.resolution.clone(),
-                              "{res_icon}" => self.shared_config.get_icon("resolution").unwrap_or_default().trim().to_string());
-
-            self.text.set_icon("xrandr");
+            let values = map!(
+                "display" => Value::from_string(m.name.clone()),
+                "brightness" => Value::from_integer(m.brightness as i64).percents(),
+                "brightness_icon" => Value::from_string(self.shared_config.get_icon("backlight_full")?), //FIXME?
```

Comment:
What's with the FIXME? Because the icon doesn't change like the backlight block?

Reply:
Yes

Reply:
Oh OK. Can you update the comment to make it more obvious? Then we can just leave it for the future.